htdocs/js/loris-scripts.js

```diff
@@ -0,0 +1,37 @@
+/* eslint new-cap: ["error", {capIsNewExceptions: ["DynamicTable", "FileUpload"]}]*/
+
+$(document).ready(function() {
```

Comment:
This snippet of code was in htdocs/js/helpHandler.js (delaeted). I migrated it in another file since it is not doing anything related to the help widget.